Time section: wording clarified, urls updated (#11979)
home-assistant/home-assistant.github.io: source/_docs/configuration/templating.markdown

### Time

> Note: UNIX timestamp is the number of seconds that have elapsed since the Unix epoch and therefore can be substituted with `int` or `float` value when used as an argument of functions below.

Reviewer: I don't think this addition makes it any clearer for users.
Please reformulate in a way, less technical users can understand it as well.

(And add a new line after it, before starting a list in Markdown).

Author: Actually, this addition is not aimed at less technical users so they can skip it without any harm.
I added it instead of writing something like this
```
- Filter `timestamp_local` converts an UNIX timestamp/float/int to a string
```
in at least 3 places.
What do you think?

Reviewer: > Actually, this addition is not aimed at less technical users

Well, we should. Hence, my request.

Author: reworded it and added a link/example

- `now()` returns a datetime object that represents the current time in your time zone.
  - You can also use: `now().second`, `now().minute`, `now().hour`, `now().day`, `now().month`, `now().year`, `now().weekday()` and `now().isoweekday()` and other [`datetime`](https://docs.python.org/3.8/library/datetime.html#datetime.datetime) attributes and functions.

> - `utcnow()` returns a datetime object that represents UTC time.

Reviewer: Timezone is not represented.

```suggestion
- `utcnow()` returns a datetime object of the current time in the UTC timezone.
```

  - For specific values: `utcnow().second`, `utcnow().minute`, `utcnow().hour`, `utcnow().day`, `utcnow().month`, `utcnow().year`, `utcnow().weekday()` and `utcnow().isoweekday()`.
- `as_timestamp()` converts datetime object or string to UNIX timestamp. This function also be used as a filter.
- `strptime(string, format)` parses a string based on a [format](https://docs.python.org/3.8/library/datetime.html#strftime-and-strptime-behavior) and returns a datetime object.
- `relative_time` converts datetime object to its human-friendly "age" string. The age can be in second, minute, hour, day, month or year (but only the biggest unit is considered, e.g. if it's 2 days and 3 hours, "2 days" will be returned). Note that it only works for dates _in the past_.

> - Filter `timestamp_local` converts an UNIX timestamp to a string that represents local time/date.

Reviewer: ```suggestion
- Filter `timestamp_local` converts an UNIX timestamp to a string in your local timezone.
```

Author: have a look at my changes

> - Filter `timestamp_utc` converts a UNIX timestamp into a string that represents UTC time/date.

Reviewer: ```suggestion
- Filter `timestamp_utc` converts a UNIX timestamp into a string in the UTC timezone.
```

Author: have a look at my changes
